TeamTracker/app/src/main/java/com/example/teamtracker/fragments/TimerFragment.java

```diff
@@ -77,23 +76,26 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean isRunning) {
     }
 
     private void showAddTaskDialog(Context context, Long duration) {
-        final EditText taskNameEditText = new EditText(context);
+        final View customLayout = getLayoutInflater().inflate(R.layout.custom_add_task_alert_dialog, null);
         AlertDialog dialog = new AlertDialog.Builder(context)
                 .setTitle("Hey you've worked for "+formatDuration(duration)+" !!")
                 .setMessage("What were you doing?")
-                .setView(taskNameEditText)
+                .setView(customLayout)
                 .setPositiveButton("Add Task", new DialogInterface.OnClickListener() {
                     @Override
                     public void onClick(DialogInterface dialog, int which) {
-                        Toast.makeText(context, "Task Created Successfully!", Toast.LENGTH_SHORT).show();
-                        Task task = new Task(String.valueOf(taskNameEditText.getText()),"", String.valueOf(project.getId()));
+                        taskTitle = customLayout.findViewById(R.id.task_Title);
+                        taskDescription = customLayout.findViewById(R.id.task_Description);
+                        Task task = new Task(String.valueOf(taskTitle.getText()),String.valueOf(taskDescription.getText()), String.valueOf(project.getId()));
```

Comment:
better you take 
String title = taskTitleEditText.getText();
String description = taskDescriptionEditText.getText();

Then,
Task task = new Task(title, description, project.getId());

it will increase readability, I believe.